pubspec.yaml

```diff
@@ -27,7 +27,10 @@ dev_dependencies:
   dart_style: ^3.0.0
   lints: ">=5.0.0 <7.0.0"
   matcher: ^0.12.15
+  puppeteer: ^3.18.0
```

Comment:
We might need to discuss if we want a non-dart-team dependency here (even if it is a dev-dependency)
cc @szakarias wdyt?

Reply:
I was thinking to implement this as an external screenshotting tool, mostly because there is no natural integration point with browser-based tests like in `pub-dev`'s integration tests, and maybe `pub-dev` could reuse the tool as a library. wdyt?